packages/design-system/src/molecules/accountInfoDropdown/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { FlatList, Pressable, StyleSheet, View } from 'react-native'
+import { CircleImg, Icon, IconVariant, TypographyV2 } from '../../atoms'
+import { useThemeV2 } from '../../hooks'
+import { Dropdown } from '../dropdown'
+import { container, margin, padding } from '../../styles'
+
+const AVATAR_SIZE = 40
+const DROPDOWN_WIDTH = 282
+
+export interface AccountInfoDropdownProps {
+  avatar: string
+  name: string
+  username: string
+  children: React.ReactNode
+  options: {
+    id: string
+    iconVariant: IconVariant
+    color?: string
+    label: string
+    onPress: () => void
+  }[]
```

Comment:
Let's extract into a separate type for a better readability

```suggestion
  options: AccountInfoDropdownOption[]
```